packages/cli/src/index.ts

```diff
@@ -183,8 +136,6 @@ async function run() {
 }
```

Comment:
because we're now not exiting with non-0 code on handleError, `run` function should call it in catch handler

Reply:
that's actually my mistake, better to return from there because `handleError` is called from different places too.